packages/ibm-products/src/components/Datagrid/utils/DatagridActions.js

```diff
@@ -178,7 +178,6 @@ export const DatagridActions = (datagridState) => {
         <RowSizeDropdown {...rowSizeDropdownProps} />
         <MenuButton
           label="Primary button"
-          renderIcon={ChevronDown}
```

Comment:
Removed.

This should _not_ throw an error. But, by default, `Menubutton's` default value for `renderIcon` is already `ChevronDown`, so removing it doesn't change the render.